src/typings.ts

```diff
@@ -0,0 +1,14 @@
+export interface Options {
+    filename?: string;
+    fieldSeparator?: string;
+    quoteStrings?: string;
+    decimalSeparator?: string;
+    showLabels?: boolean;
+    showTitle?: boolean;
+    title?: string;
+    useTextFile?: boolean,
+    useBom?: boolean;
+    valueFallback?: string;
+    headers?: Record<string, string>;
+    keys?: string[]; //This will be the columns order
```

Comment:
Isn't 'keys' mandatory?

Reply:
The comment is misleading. Yes, 'keys' is **also** used for defining the order of the columns, but I would first describe its main purpose.